### ISSUE TITLE
Dependency error with `openai`

### ISSUE THREAD

**Issue by cyai** [PrefectHQ/marvin#723]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to try to find a similar issue and didn't find one.
- [X] I searched the Marvin documentation for this issue.

### Bug summary

I am getting the error ```bash ImportError: cannot import name 'OpenAI' from 'openai' ``` when trying to run a simple marvin ai function.

When tried to fix this using `pip install openai --upgrade` I get the error:
` ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
marvin 1.5.6 requires openai<1.0.0,>=0.27.8, but you have openai 1.6.1 which is incompatible.
`

### Reproduction

```python3
My code: 

from marvin import ai_model
from pydantic import BaseModel, Field
from openai import OpenAI
import os
from dotenv import load_dotenv

load_dotenv()

client = OpenAI(api_key=os.getenv('OPENAI_API_KEY))


class Location(BaseModel):
    city: str
    state_abbreviation: str = Field(
        ..., description="The two-letter state abbreviation"
    )


ai_model(Location, client=client)("The Big Apple")



When running this using `python marvin_scirpt.py` I am getting the mentioned error.
```

### Versions

```Text
Version:		1.5.6
Python version:		3.11.4
OS/Arch:		darwin/arm64
```

**Comment by zzstoatzz** [comment]
hi @cyai - this is our fault since our docs suggest 1.5.6 has 2.x syntax

please install from main instead
```
pip install git+https://github.com/PrefectHQ/marvin.git
```

**Comment by zzstoatzz** [comment]
hi @cyai - we've released marvin 2.1, so `pip install -U marvin` to upgrade and the docs should now be in line with that version :)

**Comment by zzstoatzz** [comment]
This is a test comment.
